roles/openshift_health_checker/openshift_checks/logging/podlogs.py

```diff
@@ -0,0 +1,135 @@
+"""
+Module for ensuring logs from pods can be queried in a reasonable amount of time.
+"""
+
+import json
+import time
+
+from uuid import uuid4
+
+from openshift_checks import get_var, OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+
+
+ES_CMD_TIMEOUT_SECONDS = 30
```

Comment:
This is the source of uncertainty and flakiness of this check :-/

Monitoring the time it takes over time would be much better than a point-in-time check. What is "reasonable amount of time" for one cluster is not for another.

This could be user configurable, but then I also would not like us to end up with too many flags for people to tune.